web/src/user_profile.js

```diff
@@ -695,6 +760,74 @@ function get_human_profile_data(fields_user_pills) {
     return new_profile_data;
 }
 
+let fields_user_pills;
+
+function check_field_changed(user_id) {
+    const person = people.maybe_get_user_by_id(user_id);
```

Comment:
Can we do something like [this](https://github.com/zulip/zulip/blob/6fb1af00f17aa395fa738e5e5ef75c206745e362/web/src/dialog_widget.ts#L170) here? Maybe we can extract a common function to be used here and in `dialog_widget`. Also would be good to have a separate commit for adding code to disable the button and another commit to update the feedback.